Library/Homebrew/list.sh

```diff
@@ -4,32 +4,38 @@
 homebrew-list() {
   case "$1" in
     # check we actually have list and not e.g. listsomething
-    list) ;;
-    list*) return 1 ;;
+    list | ls) ;;
+    list* | ls*) return 1 ;;
     *) ;;
   esac
 
   local ls_args=()
   local formula=""
   local cask=""
 
-  local first_arg
-  for arg in "$@"
+  local OPTIND=2 # skip $1 (and localise OPTIND to this function)
```

Comment:
This warrants a longer comment explaining:
- what `OPTIND=2` is
- why it is needed

Reply:
```suggestion
  # `OPTIND` is used internally by `getopts` to track parsing position
  local OPTIND=2 # skip $1 (and localise OPTIND to this function)
```

maybe?

Reply:
Yup, looks great 👍🏻